benchmark/buffers/buffer-read.js

```diff
@@ -30,14 +30,14 @@ function main(conf) {
   var len = +conf.millions * 1e6;
   var clazz = conf.buf === 'fast' ? Buffer : require('buffer').SlowBuffer;
   var buff = new clazz(8);
-  var fn = 'read' + conf.type;
+  var fn = `read${conf.type}`;
```

Comment:
Yes, but there were many other template literals in benchmarks before this edition.